### PR TITLE
style(checkbox): update checkbox to sip-34 design

### PR DESCRIPTION
**Pull request by pkdotson**
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

**before** 

IGNORE NULL LOCATIONS
IGNORE NULL LOCATIONS

**after**

_ IGNORE NULL LOCATIONS
IGNORE NULL LOCATIONS
Configure filter scopes
also copy (duplicate) charts

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/components/Checkbox.jsx [hunk 1]
```diff
@@ -32,15 +32,17 @@ export default function Checkbox({ checked, onChange, style }) {
...
         onClick={() => {
           onChange(!checked);
         }}
         style={{
-          border: '1px solid #aaa',
+          border: `${checked ? '1px solid #20A7C9' : '1px solid #aaa'}`,
           borderRadius: '2px',
           cursor: 'pointer',
+          boxShaddow: 'inset 1px 0px 0px #E0E0E0',
+          backgroundColor: `${checked ? '#20A7C9' : 'transparent'}`
```
Comment (nytai) [inline superset-frontend/src/components/Checkbox.jsx:45]: I didn't realize this component exists. Thoughts on deprecating this in favor of https://github.com/preset-io/incubator-superset/blob/master/superset-frontend/src/components/IndeterminateCheckbox.tsx which is using the new Icon component and emotion styles?

Comment (pkdotson) [reply superset-frontend/src/components/Checkbox.jsx:45]: sounds like a good idea to me. @rusackas any thoughts on this?

Comment (rusackas) [reply superset-frontend/src/components/Checkbox.jsx:45]: I'm all for consolidation! Use our Checkbox wherever checkboxes are used!

Comment (mistercrunch) [reply superset-frontend/src/components/Checkbox.jsx:45]: Also let's never use hardcoded colors!

**Comment by mistercrunch** [comment]
Oh - side note - I noticed some rogue checkboxes in the "Filter Scope Modal" today that use a hard coded older theme color.
Screen Shot 2020-07-23 at 10 27 43 PM

**Pull request closed without merging**